### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by Mingling94**
- Changed travis line for virtual display
- Load confi.json via require instead of fs
- bumped electron version
- Electron's module structure changed, updated require statements
- Misc other fixes detailed in commit headlines

### CODE REVIEW

**Comment by Mingling94** [comment]
@lukechampine @DavidVorick boop

File: package.json [hunk 1]
```diff
@@ -24,20 +24,20 @@
...
-    "release-lin64": "npm run release-args -- --platform=linux --arch=x64",
-    "release-lin32": "npm run release-args -- --platform=linux --arch=ia32",
-    "release-win64": "npm run release-args -- --platform=win32 --arch=x64",
-    "release-win32": "npm run release-args -- --platform=win32 --arch=ia32",
-    "release-mac": "npm run release-args -- --platform=darwin --arch=x64"
+    "release-lin64": "npm run release-args -- --platform=linux --arch=x64 && ./.finalizeRelease.sh",
+    "release-lin32": "npm run release-args -- --platform=linux --arch=ia32 && ./.finalizeRelease.sh",
+    "release-win64": "npm run release-args -- --platform=win32 --arch=x64 && ./.finalizeRelease.sh",
+    "release-win32": "npm run release-args -- --platform=win32 --arch=ia32 && ./.finalizeRelease.sh",
+    "release-mac": "npm run release-args -- --platform=darwin --arch=x64 && ./.finalizeRelease.sh"
```
Comment (DavidVorick) [inline package.json:40]: can you explain this?

Comment (Mingling94) [inline package.json:40]: takes out `default_app` and zips up the release for each of the individual release commands.

**Comment by Mingling94** [comment]
can this be merged now? @lukechampine @DavidVorick

**Pull request merged**